content/en/apps/reference/app-settings/user-permissions.md

```diff
@@ -10,15 +10,17 @@ relevantLinks: >
 keywords: user-roles user-permissions
 ---
 
-Permissions are defined by the `permissions` object in the `base_settings.json` file. Permissions can also be configured using the App Management app.
-Each permission is defined as an array of user role identifiers that have the permission granted.
+Permissions are defined manually by the `permissions` object in the `base_settings.json` file. Each permission is defined as an array of user role identifiers that have the permission granted.
```

Comment:
@michaelkohn I've added this clarification

Reply:
How about something like this?

> Permissions are defined by the `permissions` object in the `base_settings.json` file. The list below illustrates the available system defined permissions. To utilize a permission, you will need to first add the permission as a property of the `permissions` object, and then associate the permission to  user role(s).  
> 
> Permissions can be assigned to user roles either directly in `base_settings.json` as an array of user role identifiers, or configured in the App Management app. 
> 

And at the end we can put a [code sample at the end like we do in other areas](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/patient_reports/#code-sample)?


> ```JSON
> "permissions": {
>    "can_view_reports": [ "chw_role", "supervisor_role" ],
>    ....
>    ....
> }
> ```
>

Reply:
I like it! This is the end result:

<img width="800" alt="Screen Shot 2022-08-24 at 2 58 49 pm" src="https://user-images.githubusercontent.com/66472237/186363614-499bfcd1-7d9d-480b-870e-25c76ac09693.png">
<img width="800" alt="Screen Shot 2022-08-24 at 2 59 05 pm" src="https://user-images.githubusercontent.com/66472237/186363666-5b4ada75-ffcf-468f-bfc7-d7eaaa2a26af.png">

Please let me know your thoughts.

Reply:
LGTM, thanks for updating that!